run_tests.sh

```diff
@@ -14,3 +14,8 @@ poetry run python -m unittest discover osv/ecosystems/ "*_test.py" .
 
 # Run the validation for the go/python datastore models
 cd ./go/models/internal/validate/ && ./run_validate.sh
+
+# Run the API tester
+url="localhost:${DATASTORE_EMULATOR_PORT}"
```

Comment:
I think we might as well have the `apitester` itself handle this, or at least have a port variable, but I want to get this working first before doing any cleanup